tests/visualization/test_graphviz.py

```diff
@@ -14,8 +14,9 @@
 import subprocess
 import tempfile
 import unittest
-
+import random
 import rustworkx
+import pydot
```

Comment:
You don’t want to use pydot as a dependency. We have stopped using it a year ago